src/nodeDebugConfigurationProvider.ts

```diff
@@ -93,6 +93,22 @@ export class NodeDebugConfigurationProvider extends BaseConfigurationProvider<An
       if (config.console === 'integratedTerminal' && !config.internalConsoleOptions) {
         config.internalConsoleOptions = 'neverOpen';
       }
+
+      // remove manual --inspect flags, which are no longer needed and interfere
+      if (config.runtimeArgs) {
+        const resolved: string[] = [];
+        for (const arg of config.runtimeArgs) {
+          if (arg.startsWith('--inspect-brk')) {
+            config.stopOnEntry = config.stopOnEntry ?? true;
+          } else if (arg.startsWith('--inspect')) {
```

Comment:
Maybe make these exact matches rather than `startsWith` because people might be matching passing flags to their own script or something like that.

Reply:
The CLI allows `--inspect[=adddress:port]` but I can regex that :)